gradle.properties

```diff
@@ -26,7 +26,7 @@ options.forkOptions.memoryMaximumSize=3g
 systemProp.org.gradle.dependency.duplicate.project.detection=false
 
 # Enforce the build to fail on deprecated gradle api usage
-systemProp.org.gradle.warning.mode=fail
+systemProp.org.gradle.warning.mode=all
```

Comment:
This is only a temporary work around to:

```
> Configure project :server
Declaring dependencies using multi-string notation has been deprecated. This will fail with an error in Gradle 10. Please use single-string notation instead: "com.google.protobuf:protoc:3.25.8:osx-aarch_64@exe". Consult the upgrading guide for further information: https://docs.gradle.org/9.1.0/userguide/upgrading_version_9.html#dependency_multi_string_notation
```

Not sure why that's happening. I think it might be coming from something the Gradle protobuf plugin is doing, but I'm not sure.